usage/app/lib/ContentPollStream.scala

```diff
@@ -107,8 +115,28 @@ trait ContentPollStream {
   }
 
   val pollInterval  = Duration(pollIntervalInSeconds, SECONDS)
-  val rawObservable =  Observable.interval(pollInterval)
-    .flatMap(_ => getContent)
+  case class PollAttempt(lastFail: Throwable, i: Int)
+
+  lazy val retryingPollObservable = Observable.interval(pollInterval)
+    .flatMap(_ => getContent).retryWhen(
+      _.zipWith(Observable.from(1 to maxRetries))((e: Throwable, i: Int) => PollAttempt(e,i))
+      .flatMap(attempt => {
+        val isLastAttempt = attempt.i >= maxRetries
+        val waitSeconds = pow((attempt.i),2).second
```

Comment:
Might be worth dropping a small comment explaining that this is exponential backoff in case it's not immediately obvious :-)